CONTRIBUTING.md

```diff
@@ -1,29 +1,30 @@
 # Konflux-CI documentation contributing guide
 
-## Contributing
-
-For active maintenance/development of the documentation, developers
-should create a
-[fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo)
-of the repository. Updates are performed in the forked
-repository and, when changes are ready for review, submitted as
-[pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
-to the upstream repository. Developers are encouraged to create task
-specific branches in their forked repositories, but merge request should
-always target the default branch of the
-[Konflux repository](https://github.com/konflux-ci/docs).
+There are 2 different ways you can contribute to the {ProductName} documentation:
+
+* **UI**: You can select **Edit this Page** in the banner on any page of the [Konflux documentation](https://konflux-ci.dev/docs) and suggest your changes in the web editor.
+
+* **GitHub**: You can fork the [Konflux-CI/docs repository](https://github.com/konflux-ci/docs) on GitHub and edit documentation locally on your machine. 
+
+## Contributing with a PR
+
+When contributing to documentation, we recommend that you:
```

Comment:
As a reader, at this point, I'm wondering, "What about the UI option you just mentioned? Is that not recommended?"

So I'd suggest just making some change to specify that you're talking about the recommended way to contribute _via your own fork_. Maybe you could change line 9 to read "Contributing with your own fork"?

Reply:
Great idea, thank you.
I was looking for wording that'll help differentiate between the two options and highlight the benefits of the fork option (because technically the UI option opens a PR too).